### ISSUE TITLE
Update DM file import function

### ISSUE THREAD

**Issue by icbicket** [icbicket/DataAnalysis#39]
Old DM file import function uses dm3_lib, not sure if this external library is up to date with DM4. Choose an external library and update import functions to match the output.

**Comment by slimpotatoes** [comment]
You're right it is not working. I think you can use the ncempy library ==> https://github.com/ercius/openNCEM

**Comment by icbicket** [comment]
Yes, I'll switch to ncempy. I need to figure out if they have metadata in there or not, and do some thinking about the energy calibration. I think they give the energy values directly, rather than the offset and dispersion that DM uses. And what happens if the spectrum in DM is not calibrated? I think I should use the DM calibration to import, but then let the user re-calibrate if needed.

**Comment by slimpotatoes** [comment]
Ah good point.
I can confirm at least that ncempy.io.dm.dmReader() returns data with some metadata. I already used the pixel size metadata carrying both the numbers and the units and it was ok, but I am not sure how the energy calibration is handled. Having an interface to manually recalibrate the data if needed is a definitely good idea :)

**Comment by icbicket** [comment]
Yes, I've seen that they have things like pixelSize and pixelUnits, but it also drops a lot of metadata and I've found it useful in the past to have, for example, the coordinates of DM annotations on an image, and I'm not sure if ncempy provides access to that or not.

Maybe manual recalibration should become a separate issue though, separate from this one.
